source/_docs/changing-edu-plan.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Managing Site Plan Changes and Chargeback for EDU+ Sites
+description: Learn how to manage site plan changes and chargebacks for EDU+ sites.
+categories: [managing]
+tags: [platform, organizations]
+keywords: pantheon, pantheon for agencies, org, organization, edu, edu+
+---
+Pantheon for EDU+ allows Organization Administrators to manage who can change service level [plans](https://pantheon.io/pricing) for site's within their organization. When a site is not associated with an organization, you can add a credit card to change the site's plan. When a site is associated with an EDU+ Organization, billing is managed through a contract with Pantheon, and only Organization Administrators can change a site's plan.
+
+## Specify Take Live/Payment Instructions
+From the Organization Dashboard, go to **Settings** and enter a **Take Live/Payment Instructions URL** and a **Service Level Change Instructions URL** that will be shown to users who are not Organization Administrators in the site's **Settings** area.
+
+These settings allow a payment flow so your EDU+ organization can collect information needed to take the site live, manage accounting, and chargebacks to the appropriate department or unit within your EDU.
+
+The URL will include information that you can use to auto-populate a form you control. For example:
+```
+https://www.awesome.edu/technology/web/payment-form?site_name=my-site&id=6b23f58b-132b-4547-93bd-71bb34fc0b1d&service_level=free&owner=Jane%Smith&email=jane%40awesome.edu
```

Comment:
@rachelwhitton  This seems to have some odd syntax highlighing...is there a way to disable it?